### ISSUE TITLE
[BlazorSDK]: Publishing a Blazor hosted app with a web.config results in error on duplicate publish files

### ISSUE THREAD

**Issue by pranavkm** [dotnet/aspnetcore#30268]
This is a variant of https://github.com/dotnet/aspnetcore/issues/29524 but caused entirely by the BlazorSDK. If the Server in a hosted app has a web.config, publishing with a 6.0 SDK results in the duplicate file publish issue.

**Comment by javiercn** [comment]
I think this is different. I believe this is an issue in Blazor webassembly including the web.config in this scenario, isn't it?

**Comment by pmeems** [comment]
I think I have the same problem. I have a Blazor WASM .NET5 application that published fine, without any warnings using VS2019 Preview version of 4 weeks ago.
After that publish I've updated VS2019 preview and installed VS2022 v17.0.0 preview 2. One of them also installed .NET 6 (I think VS2022) and now I can't publish my project anymore.
I get this error:
`Found multiple publish output files with the same relative path: C:\Program Files\dotnet\sdk\6.0.100-preview.6.21355.2\Sdks\Microsoft.NET.Sdk.BlazorWebAssembly\targets\BlazorWasm.web.config, \MyProject.Server\web.config.
`
I checked I don't have a second web.config and my project is targeting .NET 5.
So I'm not sure why I get this .NET 6 error.

In https://github.com/dotnet/aspnetcore/issues/29524#issuecomment-772705034 is suggested to pass `/p:ErrorOnDuplicatePublishOutputFiles=false` to the` dotnet publish` command.
Sadly enough I can't find this in the GUI of VS, meaning I need to use the command line.
But this is hiding the warning and I want to know what is wrong with my project that is causing the error.
